components/scream/data/scream_input.yaml

```diff
@@ -61,31 +77,32 @@ HOMME:
     partmethod        : 4
     cubed_sphere_map  : 0
     topology          : cube
-    test_case         : jw_baroclinic
     u_perturb         : 1
     rotate_grid       : 0
     ne                : 4
     statefreq         : 9999
     mesh_file         : /dev/null
     tstep             : 300
-    nu                : 7.0e+15
-    nu_div            : 1.0e+15
-    nu_p              : 7.0e+15
-    nu_q              : 7.0e+15
-    nu_s              : -1
-    nu_top            : 2.5e+5
+    nu                :  .000000034
```

Comment:
I think this should use scientific notation.

Reply:
I've had limited success with scientific notation and the yaml parser.  Sometimes it grabs it as a string, maybe when we use `e-X`, maybe the `-` throws it off for some reason.  I'll have to check.

Reply:
Our YAML parser should not have issues digesting scientific format.